psi4/src/psi4/libscf_solver/CMakeLists.txt

```diff
@@ -9,10 +9,11 @@ set(sources_list uhf.cc
                  stability.cc
 )
 psi4_add_module(lib scf_solver sources_list mints diis efp_solver fock)
-if(TARGET PCMSolver::PCMSolver)
+if(TARGET PCMSolver::pcm)
```

Comment:
I had to change the way the output from PCMSolver is handed over to the host. Instead of a free function that the API expects to be defined somewhere in the host, the host needs to pass this function explicitly to the module, i.e. a function pointer. I will fix this problem in your fork, the fix should do away with the compilation error you report **and** the need for the workaround in this file.